IPython/core/usage.py

```diff
@@ -76,6 +76,22 @@
 At your system command line, type 'ipython -h' to see the command line
 options available. This document only describes interactive features.
 
+GETTING HELP
+------------
+
+Within IPython you have various way to access help:
+
+  ?         -> Introduction and overview of IPython's features (this screen).
+
```

Comment:
I would remove these blank lines in between each one of these, like it used to be in the old banner.  I don't think we need all the extra whitespace here.